VideoLocker/build.gradle

```diff
@@ -256,6 +258,7 @@ android.applicationVariants.all { variant ->
         def config = taskHelper.loadConfig(project)
         helper.saveProcessedConfig(project, config)
         helper.saveResources(project, config)
+        android.buildTypes.debug.testCoverageEnabled = taskName.contains("Coverage")
```

Comment:
This causes test coverage to be enabled only if we are running a coverage test such as `testDebugUnitTestCoverage`.

Enabling test coverage breaks the debugger (causes variables to not appear & line numbers to mismatch) and there is no reason to enable it except if we are actually generating a coverage report.